DTS-STN/senior-journey
# Left Menu - Filter Results

## [ADO-147](https://dev.azure.com/JourneyLab/SeniorsJourney/_sprints/taskboard/DTS/SeniorsJourney/DTS/DTS%20-%20Sprint%205?workitem=147)

### Description

List of proposed changes:
- filter added

### What to test for/How to test
- navigate to http://localhost:3000/en/quiz/tasks
- click the filters and observe how tasks are rendered depending on what tag(s) are selected
- test responsiveness of filter at different viewport widths
- test keyboard accessibility (all interactive elements should be keyboard accessible)
